### DISCUSSION TITLE
Crediting RmlUi

### DISCUSSION THREAD

**Discussion by Paril** [mikke89/RmlUi#654] [General]
Hey, this is specifically a question for @mikke89 - I couldn't find a public email so I figured I'd just pop this here since it's not a private matter by any means, heh.

We want to credit you on our upcoming releases for RmlUi and the help you've given us (NightDive Studios) via discussions on here - how would you like to be credited? If you prefer to take the discussion private you can send me an email at paril@nightdivestudios.com and we can go from there, too.

Thanks!

**Comment by mikke89** [comment]
Hey, that is very nice and means a lot to me! I sent you en email regarding the crediting.

For anyone else seeing this, feel free to credit the library how you like, that would very much be appreciated! If you want to give more of a personal credit to me, then I'd be happy if you got in touch with me: michael.ragazzon@gmail.com.

Just keep in mind that the library is licensed under MIT, so the license itself should also be included somewhere in the distribution, regardless of other crediting.
